05-GitBest.Rmd

```diff
@@ -15,53 +15,58 @@ You can host multiple projects on GitHub at the same time but it would be a mist
 
```

Comment:
For the above section, perhaps mention why this is beneficial, or the problems that can arise when users don't do this?